classes/class-woothemes-sensei-certificate-templates.php

```diff
@@ -193,16 +193,15 @@ public function plugin_path() {
 	 * @param string $locate locate path
 	 * @return string the location path for the certificate template preview file
 	 */
-	function certificate_templates_locate_preview_template( $locate ) {
+	public function certificate_templates_locate_preview_template( $locate, $type, $templates ) {
```

Comment:
Thanks for the PR @bernattorras. Would you mind updating the comment for this function to include the additional params that are now being passed? 🙇🏻‍♀️

Reply:
Also, since we're not using `$templates`, maybe we can skip passing it?